src/Drawer/index.scss

```diff
@@ -12,7 +12,6 @@
   transition: transform var(--transition-speed-fast) ease;
   border: var(--border-size-s);
   border-color: var(--border-color-light);
-  padding: var(--space-xxl);
```

Comment:
my thinking here was that we always want to apply this amount of padding to the `.drawer.navigation` container. But for the actual drawer contents (the children we're passing in), we want to apply a minimum padding regardless of whether the drawer is opening from top/bottom or left/right